src/transformState.js

```diff
@@ -4,8 +4,35 @@
  * @param {Object} state
  * @param {Object[]} actions
  */
+const removeKey = (keysToRemove, data) => {
+  for (const keyToRemove of keysToRemove) {
+    delete data[keyToRemove];
+  }
+};
+const clearProperties = (action, data) => {
```

Comment:
Redundant argument  `action`, you`re not using it

Reply:
I'll put it on QnA

Reply:
fixed now